docs/SearchBar.md

```diff
@@ -37,6 +37,7 @@ Then, add `search-bar` block into your app theme, as we do in our [Store Header]
 | `openAutocompleteOnFocus` | `Boolean`                                     | Identify if autocomplete should be open on input focus or not.                                                                                                                                    | `false`        |
 | `placeholder`             | `String!`                                     | Placeholder to be used on the input                                                                                                                                                               | -              |
 | `inputType` | `enum` | Defines the value for the `type` HTML attribute (from the `<input>` field). Possible values are: `search` and `text`. We strongly recommend you to use `search` as it fixes some iOS issues. The `text` value is only kept as default for backwards compatibility. | `text` |
+| `containerMode` | `String` | Allows the autocomplete component to be displayed with or without overlay
```

Comment:
Which are the possible values for this prop?

Reply:
There are currently two props, overlay and container, but I just updated the readme : )